process/block/metablock.go

```diff
@@ -1520,6 +1527,106 @@ func (mp *metaProcessor) updateState(metaBlock data.MetaHeaderHandler, metaBlock
 	mp.blockChain.SetFinalBlockInfo(metaBlock.GetNonce(), metaBlockHash, metaBlock.GetRootHash())
 }
 
+func (mp *metaProcessor) pruneTriesHeaderV3(
+	metaBlock data.MetaHeaderHandler,
+	prevMetaBlock data.MetaHeaderHandler,
+	prevMetaBlockHash []byte,
+) {
+	accountsDb := mp.accountsDB[state.UserAccountsState]
```

Comment:
same as for shard, can this be affected by a parallel execution using the same DB but on a different roothash (or at most starting at metablock.LastExecutionResult-> roothash)